plugins/woocommerce-blocks/packages/components/totals/item/index.tsx

```diff
@@ -39,7 +39,9 @@ const TotalsItemValue = ( {
 			currency={ currency || {} }
 			value={ value as number }
 		/>
-	) : null;
+	) : (
+		<>-</>
```

Comment:
Similar to the other part, I mentioned before, this part also looks a bit odd. What about refactoring this part to ensure that the expected return happens at the bottom? Something like this:

```ts
	if ( ! Number.isFinite( value ) ) {
		return <>-</>;
	}

	return (
		<FormattedMonetaryAmount
			className="wc-block-components-totals-item__value"
			currency={ currency || {} }
			value={ value as number }
		/>
	);
```

To me, the code above feels a bit more readable than the current one:

```ts
	return Number.isFinite( value ) ? (
		<FormattedMonetaryAmount
			className="wc-block-components-totals-item__value"
			currency={ currency || {} }
			value={ value as number }
		/>
	) : (
		<>-</>
	);
```